README.md

```diff
@@ -157,6 +157,12 @@ public function forgetAllTranslations(string $locale)
 public function getTranslations(string $attributeName): array
 ```
 
+#### Getting the specified translations in one go
+
+``` php
```

Comment:
In this readme, we don't just offer the function signature, but always give a nice description and example